src/Operation/Find.php

```diff
@@ -78,6 +78,9 @@ class Find implements Executable
      *
      *  * max (document): The exclusive upper bound for a specific index.
      *
+     *  * maxAwaitTimeMS (integer): The maxium amount of time for the server to wait
+     *    on new documents to satisfy a query, if “tailable” and “awaitData” are both true.
```

Comment:
Although the extension does support `tailable` and `awaitData` options on [`MongoDB\Driver\Query`](http://php.net/manual/en/mongodb-driver-query.construct.php), PHPLIB implements the [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst) which exposes those through the `cursorType` enum. In this case, we can say "if cursorType is TAILABLE_AWAIT".

Side note: try to keep curly quotes out of source files if possible. I assume these only snuck in by copying text from the PHP.net docs.